viz-lib/src/components/visualizations/editor/createTabbedEditor.jsx

```diff
@@ -33,7 +35,7 @@ TabbedEditor.propTypes = {
   tabs: PropTypes.arrayOf(
     PropTypes.shape({
       key: PropTypes.string.isRequired,
-      title: PropTypes.string.isRequired,
+      title: PropTypes.oneOfType([PropTypes.string, PropTypes.func]).isRequired,
```

Comment:
```suggestion
      title: PropTypes.oneOfType([
        PropTypes.string,
        PropTypes.func, // (options) => string
      ]).isRequired,
```

Just to keep some "context" attached to the prop.